Elevel/Application/Elevel.Application.csproj

```diff
@@ -4,9 +4,16 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
need to remove this

Reply:
I thought someone would need that, so I kept it alone, actually. Now it's deleted